README.md

```diff
@@ -126,7 +126,7 @@ for more information. See also the [`AUTHORS`](AUTHORS) and
 
 Effcee depends on the [RE2][RE2] regular expression library.
 
-Effcee tests depend on [Googletest][Googletest] and [Python][Python].
+Effcee tests depend on [Googletest][Googletest] and [Python 3][Python].
```

Comment:
How does this link render? Previously it was www.python.org

Reply:
You can see it at https://github.com/dneto0/effcee/tree/python3 
Python and 3 are both in blue.